### ISSUE TITLE
Feature Request: Open tabs in new or existing groups

### ISSUE THREAD

**Issue by Bamajon1974** [nkrishnaswami/open-selected-links#2]
Request to "open selected links" in new or existing groups in addition to same or new window. Thanks!

**Comment by nkrishnaswami** [comment]
This should work as of the latest commit, 77024e1.  This also switches to MV3 (for reasons), so I had to add some weird special cases to support Opera (which I mostly use), and the permissions changes may make the Chrome web store review harder. But we will see!

**Comment by nkrishnaswami** [comment]
(This is currently under review as v. 1.5)

**Comment by Bamajon1974** [comment]
I downloaded the new version and neither option of open selected links in existing nor new window work, and there is no option for opening in a new or same tab group. I restarted Chrome/Edge and issue still persists. Am I doing something wrong?

**Comment by nkrishnaswami** [comment]
The tab group UI is in the extension's popup; I have it pinned in my browser to make that easier to get at. That seems to work consistently.
Is the not working you see for the context menu options? I have started getting that intermittently now too.... let's make a new issue for that.
(I've just submitted updated screenshots to the Chrome Web Store.)

**Comment by Bamajon1974** [comment]
Ohhh, I found the tab group UI. I normally hide most extensions in the extensions menu to prevent clutter, that's why I didn't notice. 

As far selecting and opening links, neither right-click option works. Something broke in the last update. Is there more information I can send to help?

**Comment by nkrishnaswami** [comment]
I found it -- it worked for me since I had moved the installing the listener for the context menu items into the installed listener, and I had just installed. Moving that back to the top level seems to have fixed it.  I committed that as 1.5.1, but have to wait for my updated screenshots to be approved before I can upload a new version to the chrome web store. 😬 
While I was thinking about it, I added a context menu option to open things in the current window in a fresh tab group, to better support use cases like yours. I've committed that as 1.6.0 and will upload instead of 1.5.1, once I can submit a new version.
